marlowe-dashboard-client/src/Component/Contacts/State.purs

```diff
@@ -172,6 +167,24 @@ handleAction (ClipboardAction clipboardAction) = do
   mapAction ClipboardAction $ Clipboard.handleAction clipboardAction
   addToast $ successToast "Copied to clipboard"
 
```

Comment:
Understanding check: This is just a plain refactoring - replacing an action which was exclusively used by other handler.

Reply:
Yes, it didn't make sense to have it as an Action if it was only used internally